src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Execute.cs

```diff
@@ -20,6 +20,7 @@ internal abstract partial class AbstractOleCommandTarget
     {
         public virtual int Exec(ref Guid pguidCmdGroup, uint commandId, uint executeInformation, IntPtr pvaIn, IntPtr pvaOut)
         {
+            this.CommandID = commandId;
```

Comment:
I hate making this a field that is only valid during the call to Exec.  Just do the plumbing to push it through as a parameter.